queries/query/highlights.scm

```diff
@@ -63,6 +63,15 @@
 ((comment) @keyword.directive @nospell
   (#lua-match? @keyword.directive "^;+%s*format%-ignore%s*$"))
 
+((predicate
+  name: (identifier) @_name
+  parameters: (parameters
+    .
+    (capture)?
+    .
```

Comment:
Is that really needed?

Reply:
It should be to cover for `(#set! foo)` and `(#set! @capture foo)`. Then the question should be: will we need to add capture for `(#set "foo")` and `(#set! @capture "foo")`

Reply:
Both work fine for me without the anchoring.

Reply:
Also, is `@property` the best capture here? It's also used for field names (like `name` in this pattern).

Reply:
Docs say that:

https://github.com/nvim-treesitter/nvim-treesitter/blob/e265fec94c7dc0c8c64cb86820ff5ad3ee135c7d/CONTRIBUTING.md?plain=1#L145

So that's the best capture that I can think for this.

Reply:
Yeah, fair. But then the field name is wrong ;)

Reply:
Maybe `@variable.member` for the field? But `@label` also sounds good.

Reply:
> The anchoring is to prevent highlighting value in (#set! @foo key value), but I don't know how common that is.

I tested that as well; `value` remains captured as `@string`.

 > Maybe @variable.member for the field? But @label also sounds good

I think the former is better; `@label` is more for GOTO and friends.

Reply:
> The anchoring is to prevent highlighting value in (#set! @foo key value), but I don't know how common that is.

Ah, you're right, you do need it for 
```
(#set! "priority" 90)
```
which is pretty common. Sorry! Can you put the anchors back then (and rename the field name)?

Reply:
It's a bit of an annoyance that the metadata can also be specified as a string. What do we want to do about that? Add an alternation, or leave it like that?

Reply:
What about capturing just the string content without quotes? Something like:

```scm
[
  (identifier) @property
  (string
    (string_content) @property)
]
```

![image](https://github.com/user-attachments/assets/43dccd8a-d8c3-47c5-8654-2f56c730e629)

Reply:
I'm half-tempted to just remove the quotes (and possibly support for the quoted form in Neovim). Two ways of doing the same thing is bad.

Reply:
Then I'll just capture `(identifier)` for now